nixos/tests/libgdata.nix

```diff
@@ -0,0 +1,23 @@
+# run installed tests
+import ./make-test.nix ({ pkgs, ... }:
+
+{
+  name = "libgdata";
+
+  meta = {
+    maintainers = pkgs.libgdata.meta.maintainers;
+  };
+
+  machine = { pkgs, ... }: {
+    environment.systemPackages = with pkgs; [ gnome-desktop-testing ];
+    # # GLib-GIO-DEBUG: _g_io_module_get_default: Found default implementation dummy (GDummyTlsBackend) for ‘gio-tls-backend’
+    # Bail out! libgdata:ERROR:../gdata/tests/common.c:134:gdata_test_init: assertion failed (child_error == NULL): TLS support is not available (g-tls-error-quark, 0)
+    services.gnome3.glib-networking.enable = true;
+    # TODO: remove this once libproxy is wrapped
+    environment.variables.XDG_DATA_DIRS = [ "${pkgs.gsettings-desktop-schemas}/share/gsettings-schemas/${pkgs.gsettings-desktop-schemas.name}" ];
```

Comment:
Can we use `glib.getSchemaPath` here?

Reply:
I would rebase this onto https://github.com/NixOS/nixpkgs/pull/67957, allowing us to drop this line.

Reply:
Ah ok, we'll merge that first then.

Reply:
Ok https://github.com/NixOS/nixpkgs/pull/67957 is merged, but this would have to go to staging.
So mental note to undo it.